packages/legacy-client/src/client/interfaces/baseObjectSet.ts

```diff
@@ -26,8 +26,8 @@ import type { ObjectSet } from "./objectSet";
 import type { SelectableProperties } from "./utils/OmitProperties";
 
 export type BaseObjectSet<O extends OntologyObject> =
-  & ObjectSet<O>
-  & BaseObjectSetOperations<O>;
+  & BaseObjectSetOperations<O>
+  & ObjectSet<O>;
```

Comment:
Changing the order here seemed to give precedence to the first `select` typings it found, which gives us the `...WithGet` return type there.